client/main.lua

```diff
@@ -21,6 +21,22 @@ exports('DespawnInterior', function(objects, cb)
     end)
 end)
 
+exports('CreateShell', function (shell, spawn, exit)
+	shell = (type(shell) == 'number' and shell) or joaat(shell)
```

Comment:
```suggestion
	shell = type(shell) == 'number' and shell or joaat(shell)
```
No need for the brackets around this, it already does the job without it